scripts/analysis.py

```diff
@@ -944,12 +912,14 @@ def multi_line_plot(dictionary, timestep,
         else:
             label = str(key)
 
-        plt.plot(init_year + (timestep/12),
+        plt.plot(init_year + (timestep / 12),
                  dictionary[key],
                  label=label)
         color_index += 1
-        ax = plt.gca()
-        ax.get_yaxis().set_major_formatter(plt.FuncFormatter(lambda x, loc: "{:,}".format(int(x))))
+        if sum(sum(dictionary[k]) for k in dictionary) > 1000:
```

Comment:
1) you probably want to give the dictionary a more meaningful name than dictionary. 
2) The proper way to sum all the values in a dictionary, d, is ```sum(d.values())```